app/assets/v2/css/vision.css

```diff
@@ -0,0 +1,156 @@
+:root {
+  --tl-color: white;
+  --tl-bg: #3E00FF;
+  --tl-date-color: var(--gc-green-hover);
+}
+
+.bg-blue {
+  background-color: var(--tl-bg);
+  color: white;
+}
+
+.vision h1 {
+  font-size: 3.2rem;
```

Comment:
Could we remove this and stick to the default it to the size used on the landing page ?

Reply:
You mean to be the default size of h1? For me is ok just is not going to be like the design file.  Just confirm and I can remove it.

Reply:
Yup ! Let's stick to the default size
We don't font inconsistent font size for diff titles across pages.